nntrainer/layers/common_properties.h

```diff
@@ -189,6 +192,39 @@ class InputSpec : public nntrainer::Property<ConnectionSpec> {
   bool isValid(const ConnectionSpec &v) const override;
 };
 
+/**
+ * @brief Padding2D property, this is used to calculate padding2D
+ * @details Padding2D is saved as a string. Upon calling Padding2D::compute,
+ * returns std::vector<unsigned int> which has computed padding2Ds, below
+ * formats are accepted valid
+ * 1. "same" (case insensitive literal string)
+ * 2. "valid" (case insensitive literal string)
+ * 3. "padding2D_all",
```

Comment:
Will these property options be open to the user? tf/pytorch does without these options, so I wonder how they do it and how compatible will be this property with their configuration.

Reply:
option 4 is to conform with tizen 6.0 abi.
And numbered padding is also supported in pytorch.

FYI) maybe my comment is a bit misleading. option 3, 4, 5 is numbers

Reply:
> FYI) maybe my comment is a bit misleading. option 3, 4, 5 is numbers

I think this has resolved my confusion.

Reply:
Updated the comment :)